### DISCUSSION TITLE
今天，IE 瀏覽器正式退役

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#454] [General]
今天，IE 瀏覽器正式退役
這個元老級的瀏覽器，真的要退役了，以前真的都用IE上網呢!!
https://tw.news.yahoo.com/%E4%BB%8A%E5%A4%A9%EF%BC%8Cie-%E7%80%8F%E8%A6%BD%E5%99%A8%E6%AD%A3%E5%BC%8F%E9%80%80%E5%BD%B9-033702490.html

**Comment by lx3211** [comment]
小時候真的很有耐性等IE在那邊慢慢跑，現在耐性越來越差了哈

**Comment by Chenszuhong** [comment]
真的是童年的回憶，但漸漸長大發現google比較多其他支援的應用程式，叫IE方便

**Comment by xCk27x** [comment]
結果新的Edge還不錯用
從google跳槽了~

**Comment by NcuJerryKao** [comment]
真的耶 已經停役了 進入IE後 它會自動跳到edge

**Comment by Ben6uo** [comment]
童年阿 想不到這一天那麼快來 不過現在有edge

**Comment by todoroki0910** [comment]
真的不禁感嘆時間過得真，不過舊的不去新的不來來

**Comment by gnsJhenJie** [comment]
覺得Edge比Chrome好用，開超多分頁(5 60個)都不太會當機。用Edge的另一個原因是不想給Google再多蒐集資料了，Google蒐集資料的程度真的太可怕了，用Edge應該可以少給他一點

**Comment by BingRong1217** [comment]
時代的眼淚...

**Comment by awkward-willy** [comment]
為了要支援 IE 有許多工程師都得特別設計與開發，現今它的退役或許能讓許多網站更新使用的技術！

**Comment by Aerofe** [comment]
個人認為edge非常好用，除了占的Ram比google少以外，google有的功能edge也都有，非常的方便！

**Comment by limitx0** [comment]
現在Edge還有IE模式
用IE模式瀏覽網頁感覺像是回到了童年

**Comment by larry40305** [comment]
新的edge也有滿多人使用的，雖然現在google還是主流

**Comment by ghost** [comment]
ie真的是時代的眼淚，還記得小時候常使用他來上網打遊戲，不過現在取代ie的edge我個人覺得比google好用，所以ie被換掉或許是個必要的發展吧

**Comment by bww0401** [comment]
IE真的成了童年回憶...，我記得以前打開電腦，第一件事就是點開IE瀏覽器，玩著那時候最流行的塞爾號還有爆爆王，真的是快樂到爆，沒想到長大之後快樂變得這麼難，IE也消失了
